.github/workflows/main.yml

```diff
@@ -480,7 +488,13 @@ jobs:
       matrix:
         os: ["windows-2022"]
         vs-toolset: ["14.1", "14.3"]
-        python-version: ["3.8", "3.10", "3.11"]
+        include:
+        - python-version: 3.8
+          fmt-ver: 10
+        - python-version: 3.10
+          fmt-ver: 8.1
+        - python-version: 3.11
+          fmt-ver: 7.1
```

Comment:
With this one, I think we still need a `python-version` list if we want it to run all the combinations of `python-version` and `vs-toolset`. In the latest CI run, this only ran (VS 14.1, Python 3.11, fmt 7.1) and (VS 14.3, Python 3.11, fmt 7.1).